File: HslCommunication_Java/HslCommunication/src/HslCommunication/Core/Net/StateOne/AlienSession.java
package HslCommunication.Core.Net.StateOne;

import java.net.Socket;

/**
 * 异形客户端的对象
 */
public class AlienSession {

    /**
     * 实例化一个默认对象
     */
    public AlienSession()
    {
        this.isStatusOk = true;
    }


    /**
     * 获取套接字
     * @return
     */
    public java.net.Socket getSocket() {
        return socket;
    }

    /**
     * 设置套接字信息
     * @param socket 当前的值
     */
    public void setSocket(java.net.Socket socket) {
        this.socket = socket;
    }

    /**
     * 获取设备唯一的DTU对象
     * @return
     */
    public String getDTU() {
        return DTU;
    }

    /**
     * 设置设备的唯一的DTU信息
     * @param DTU
     */
    public void setDTU(String DTU) {
        this.DTU = DTU;
    }

    /**
     * 获取当前的连接状态是否正常
     * @return
     */
    public boolean getIsStatusOk() {
        return this.isStatusOk;
    }

    /**
     * 设置当前的连接状态
     * @param isStatusOk
     */
    public void setIsStatusOk(boolean isStatusOk) {
        this.isStatusOk = isStatusOk;
    }




    private Socket socket = null;               // 网络套接字
    private String DTU = "";                    // 唯一的标识
    private boolean isStatusOk = false;         // 当前的网络状态

}

File: HslCommunication_Net45/Core/Net/StateOne/AlienSession.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net.Sockets;
using System.Text;

namespace HslCommunication.Core.Net
{
    /// <summary>
    /// 异形客户端的异步对象
    /// </summary>
    public class AlienSession
    {
        /// <summary>
        /// 实例化一个默认的参数
        /// </summary>
        public AlienSession()
        {
            IsStatusOk = true;
        }


        /// <summary>
        /// 网络套接字
        /// </summary>
        public Socket Socket { get; set; }

        /// <summary>
        /// 唯一的标识
        /// </summary>
        public string DTU { get; set; }

        /// <summary>
        /// 指示当前的网络状态
        /// </summary>
        public bool IsStatusOk { get; set; }
    }
}
